src/components/DatePicker/index.android.js

```diff
@@ -30,36 +42,44 @@ class DatePicker extends React.Component {
      * @param {Date} selectedDate
      */
     raiseDateChange(event, selectedDate) {
```

Comment:
I know this is not your code, but I feel like this function could use a better name, maybe `setDate` would make more sense?

Reply:
I agree, I will add it to the next commit.